runtime/calamari/src/lib.rs

```diff
@@ -696,22 +845,52 @@ pub type Barrier = (
 	AllowSubscriptionsFrom<ParentOrSiblings>,
 );
 
-pub struct XcmConfig;
-impl Config for XcmConfig {
+parameter_types! {
+	/// Xcm fees will go to the asset manager (we don't implement treasury yet)
+	pub XcmFeesAccount: AccountId = AssetManager::account_id();
```

Comment:
If fees accrue to the asset manager account (which if i see the code right is just converted from the pallet id?), who can access them in the future? It doesn't seem like asset-manager itself has a transfer extrinsic.

Is this going to be a governance force_transfer() type of tx to transfer to treasury some time later?

Reply:
good catch. Since we have treasury, it should go to treasury actually.

Reply:
Yeah the root through governance can force_transfer .
Otherwise I'm not sure if we have access to this account directly.

Reply:
Umm, @ghzlatarev I think you didn't change this to treasury before merging.
What's the reason you prefered it to stay the PalletId account?

I'm randomly looking at this again since I just found an issue by @Dengjianping describing this 
https://github.com/Manta-Network/Manta/issues/185

Reply:
I think I missed that request, can you change it in your PR https://github.com/Manta-Network/Manta/pull/485